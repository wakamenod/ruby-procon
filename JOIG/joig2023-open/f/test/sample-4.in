7
EACHBAG
5
130 104 162
107 219 45
144 168 157
213 79 257
100000000000 100000000000 100000000000
