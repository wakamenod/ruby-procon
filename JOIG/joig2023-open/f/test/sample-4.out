2078
1766
2465
2894
1600000000000
